html/Product/index.html

```diff
@@ -0,0 +1,266 @@
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+          <img src="./assets/rate-star/star.png" alt="Rate star">
+        </div>
+        <p class="rate__reviews">3 Review(s)</p>
+        <a class="rate__add-review" href="#">add review</a>
+      </div>
+
+      <!-- Price -->
+      <p class="details__price">
+        $499.00
+      </p>
+
+      <!-- Storehouse details -->
+      <div class="details__storehouse">
+        <p>
+          Availability : <span>In stock</span>
+        </p>
+        <p>
+          Product Code : <span>#4657</span>
+        </p>
+        <p>
+          Tags : <span>Fashion, Hood, Classic</span>
+        </p>
+      </div>
+
+      <!-- Product description -->
+      <div class="details__description">
+          <p class="description__text">
+            Sleek, polished, and boasting an impeccably modern fit, this blue, 2-but-
+            ton Lazio suit features a notch lapel, flap pockets, and accompanying flat
+            front trousers—all in pure wool by Vitale Barberis Canonico.
+          </p>
+          
+          <ul class="description__list">
+            <li>
+              Dark blue suit for a tone-on-tone look
+            </li>
+            <li>
+              Regular fit
+            </li>
+            <li>
+              100% Cotton
+            </li>
+            <li>
+              Free shipping with 4 days delivery
+            </li>
+          </ul>
+      </div>
+
+      <!-- Order form -->
+      <form class="details__form">
+
+        <div class="form__container">
+           <!-- Color -->
+          <div>
+            <label for="color">COLOR</label>
+            <select id="color" class="form__color">
+                <option disabled selected value>Select Color</option>
+                <option value="Black">Black</option>
+                <option value="White">White</option>
+            </select>
+          </div>
+        
+          <!-- Size -->
+          <div>
+            <label for="size">SIZE</label>
+            <select id="size" class="form__size">
+                <option disabled selected value>Select Size</option>
+                <option value="Big">Big</option>
+                <option value="Small">Small</option>
+            </select>
+          </div>
+
+          <!-- Amount -->
+          <div> 
+            <label for="amount">QTY</label>
+            <input id="amount" class="form__amount" type="number" min="1" max="10" value="1"> 
+          </div>
+        </div>
+       
+        <!-- Reset -->
+        <button type="reset" class="form__reset">
+          clear selection
+        </button>
+        
+        <div class="form__bookmarks">
+          <!-- Add to cart -->
+          <button class="bookmarks__button-1" type="submit">
```

Comment:
button-1 and button-2 bad class names, this buttons have a lot in common, different only color/background why not make something as base styles and with modifiers when needed change background and color